include/Graphs/PAG.h

```diff
@@ -607,6 +607,18 @@ class PAG : public GenericGraph<PAGNode,PAGEdge> {
     inline const MemObj* addDummyMemObj(NodeID i, const Type* type) {
         return SymbolTableInfo::Symbolnfo()->createDummyObj(i,type);
     }
+    inline NodeID addCloneDummyObjNode(const MemObj *mem) {
```

Comment:
Could you please put these three functions in your TBHC.h/cpp, so that it will not be misused by other users?